### PR TITLE
volume limits count release notes

### PR DESCRIPTION
**Pull request by multi-io**
Release notes: volume-count-limits. Will be active with 2.11

### CODE REVIEW

File: pages/01.metakube/01.release-notes/default.en.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ page-toc:
 <!-- markdownlint-disable MD036 -->
 ## July 2019
 
+**2019-07-18**
+
+* The scheduler is now aware of the maximum number of persistant storage volumes that can be attached to a node, and will try to make sure that pods with mounted volumes are only scheduled on nodes can support them.
```
Comment (bashofmann) [inline pages/01.metakube/01.release-notes/default.en.md]: And will not try to do it, it ensures that and if there is no node where there is space the pod stays in pending.
Also this affects only OpenStack and we should mention what the limit is.

**Review by bashofmann — CHANGES_REQUESTED** [review]

File: pages/01.metakube/01.release-notes/default.en.md (hunk 1 above)
Comment (multi-io) [reply pages/01.metakube/01.release-notes/default.en.md]: Updated.

File: pages/01.metakube/01.release-notes/default.en.md [hunk 2]
```diff
@@ -10,6 +10,17 @@ page-toc:
 <!-- markdownlint-disable MD036 -->
 ## July 2019
 
+**2019-07-22**
+
+* Kubernetes versions 1.14.4, 1.12.10, 1.13.8, 1.15.0 and 1.15.1 are now available
```
Comment (cbeneke) [inline pages/01.metakube/01.release-notes/default.en.md]: These should be ordered (e.g. 1.12.10, 1.13.8, 1.14.4, 1.15.0 and 1.15.1)

File: pages/01.metakube/01.release-notes/default.en.md [hunk 3]
```diff
@@ -10,6 +10,17 @@ page-toc:
...
+* Updated [Helm](https://www.helm.sh/) to [2.14.2](https://github.com/helm/helm/releases/tag/v2.14.2)
+* On Openstack, the scheduler is now aware that only 25 additional volumes may be attached to a node. It will make sure that pods with mounted volumes are only scheduled on nodes can support them.
```
Comment (cbeneke) [inline pages/01.metakube/01.release-notes/default.en.md]: `are only scheduled on nodes can support them` .. this should be `...on nodes that support them` (or alike)

**Review by cbeneke — CHANGES_REQUESTED** [review]

**Review by cbeneke — APPROVED** [review]

**Review by bashofmann — APPROVED** [review]

**Pull request merged**